Firebase/InstanceID/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-# Unreleased
+# 20210-1 - 7.0.1
```

Comment:
2021-1 - 7.4.0

Should be 7.4.0 now that we've gone to single version. You can find it by looking at the milestone selection

Reply:
Oops, also fixed the typo.